docker/Dockerfile-Jenkins-Build-Env

```diff
@@ -4,7 +4,7 @@ FROM ubuntu:18.04
 
 # Install all the pre-requisits we need in the docker image
 RUN apt-get update \
-  && apt-get install -y qemu-user-static docker.io sudo bzip2
+  && apt-get install -y qemu-user-static docker.io sudo bzip2 ccache
```

Comment:
Same here. Is `ccache` necessary?

Reply:
https://github.com/eclipse/kuksa.val/pull/42#discussion_r483722967